src/infra/raft/v2/RaftService.cpp

```diff
@@ -51,6 +51,7 @@ void RaftServer::serverLoopMain() {
   pthread_setname_np(pthread_self(), "RaftServer");
 
   grpc::ServerBuilder builder;
+  builder.SetMaxReceiveMessageSize(INT_MAX);
   builder.AddListeningPort(mIpPort, TlsUtil::buildServerCredentials(mTlsConfOpt));
```

Comment:
const auto &ipaddr = mDNSResolver.resolve(mIpPort);
RaftCore mutual tls is enabled, raftserver also needs enable parser.

Reply:
not an issue